src/common/reactNativeProjectHelper.ts

```diff
@@ -17,11 +19,13 @@ export class ReactNativeProjectHelper {
     }
 
     /**
-     * Ensures that we are in a React Native project and then executes the operation
+     * Ensures that we are in a React Native project
      * Otherwise, displays an error message banner
-     * {operation} - a function that performs the expected operation
      */
     public isReactNativeProject(): Q.Promise<boolean> {
+        if (!fs.existsSync(path.join(this.workspaceRoot, "package.json"))) {
```

Comment:
This workspaceRoot variable actually is pointing at project root. I'll rename it. Thanks for noticing this inconsistency!